aspnet/visual-studio/overview/2013/aspnet-and-web-tools-20132-preview-for-visual-studio-2013-release-notes.md

```diff
@@ -111,7 +111,7 @@ Firefox uses its own certificate store, so it will display a warning.
     ![Intellisense showing full JavaScript object](aspnet-and-web-tools-20132-preview-for-visual-studio-2013-release-notes/_static/image15.png)
 - **New URL Picker in HTML, Razor, CSS, LESS and Sass documents**: VS 2013 shipped with no URL picker outside of Web Forms pages. The new URL picker for HTML, Razor, CSS, LESS and Sass editors is a dialog-free, fluent typing picker that understands '..' and filters file lists appropriately for img tags and links.
 
-    ![URL Picker for image tag](aspnet-and-web-tools-20132-preview-for-visual-studio-2013-release-notes/_static/image16.png)    ![URL Picker for views](aspnet-and-web-tools-20132-preview-for-visual-studio-2013-release-notes/_static/image17.png)    ![URL Picker for CSS](aspnet-and-web-tools-20132-preview-for-visual-studio-2013-release-notes/_static/image18.png)
+    ![Screenshot of new dialog-free, fluent typing u r l Picker that filters file lists appropriately for i m g tags and links.](aspnet-and-web-tools-20132-preview-for-visual-studio-2013-release-notes/_static/image16.png)    ![Screenshot of new dialog-free, fluent typing u r l Picker that filters file lists appropriately, displaying h t m l links.](aspnet-and-web-tools-20132-preview-for-visual-studio-2013-release-notes/_static/image17.png)    ![Screenshot of new dialog-free, fluent typing Picker for c s s editors that filters file lists appropriately for i m g tags and links.](aspnet-and-web-tools-20132-preview-for-visual-studio-2013-release-notes/_static/image18.png)
```

Comment:
Make these images into separate lines. All these images and alt text descriptions is...kinda messy.